tests/VimPlugin/project/match_project.vader

```diff
@@ -0,0 +1,12 @@
+Include (Prepare project directories): lib/create_fixtures_directory.vader
```

Comment:
Is this test the same as `find_project.vader`?

Reply:
Indeed, both should have more expressive names. 
- `find_project` is a part of repository test, 
- `match_project` is a part bufferMatcher test.